lib/tesla/middleware/logger.ex

```diff
@@ -27,6 +27,7 @@ defmodule Tesla.Middleware.Logger.Formatter do
     Enum.map(format, &output(&1, request, response, time))
   end
 
+  defp output(:query, env, _, _), do: env.query |> inspect()
```

Comment:
We should output the same format that will actually be used by tesla - https://github.com/teamon/tesla/blob/master/lib/tesla.ex#L500

Reply:
I considered that, and wasn't sure if it made sense to make `encode_query/1` available in `Tesla.Middleware.Logger.Formatter`.

I'm happy to make that change. I'll take a second look shortly.

Reply:
Ok, I updated the PR. Let me know if there's anything else I should do.